ts/spi/provider_frame.ts

```diff
@@ -62,7 +62,6 @@ export class ProviderFrame {
           await providerConfig.clientConfigurationProvider.getConfiguration(
               providerConfig.clientAuthDomain);
       if (!clientConfiguration || !clientConfiguration.apiEnabled) {
-        console.info('OpenYOLO API is not enabled for the client origin');
```

Comment:
The error just below will be thrown and displayed in the console, so I think this is redundant.